EPI/parse-GTRI.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+if __name__ == '__main__':
+    gtri_json_filename = "models/CHIME-SIR-GTRI.json"
+    output_json_filename = "models/formatted-CHIME-SIR-GTRI.json"
```

Comment:
For the output, we could use the file name and maybe add `_formatted` at the end.

Reply:
I'm going to do the same for the GrFN python script.

Reply:
I updated the GTRI script. I let the user specify the name of the output file entirely. I can change it though.

Reply:
No, this is great. Thank you.